app/javascript/src/javascripts/post_mode_menu.js

```diff
@@ -30,29 +30,23 @@ PostModeMenu.show_notice = function (i) {
 };
 
 PostModeMenu.change_tag_script = function (e) {
-  if ($("#mode-box-mode").val() === "tag-script") {
-    const old_tag_script_id = LS.get("current_tag_script_id") || "1";
-
-    const new_tag_script_id = parseInt(e.key, 10);
-    const new_tag_script = LS.get("tag-script-" + new_tag_script_id);
+  if ($("#mode-box-mode").val() !== "tag-script")
+    return;
+  e.preventDefault();
 
-    $("#tag-script-field").val(new_tag_script);
-    LS.put("current_tag_script_id", new_tag_script_id);
-    if (old_tag_script_id !== new_tag_script_id) {
-      PostModeMenu.show_notice(new_tag_script_id);
-    }
+  const newScriptID = parseInt(e.key, 10);
```

Comment:
```suggestion
  const newScriptID = Number(e.key);
```

For base 10 `parseInt` and `Number` are roughly equivalent in functionality (assuming you don't need `parseInt`'s functionality of parsing up until the first non-number, `Number` bails out if any part of the string isn't a number), but `Number` is ~30% faster

Reply:
The performance is irrelevant, right? It only matters that we only expect numbers as input, and so Number(x) is shorter than parseInt(x, 10) ...

Reply:
> The performance is irrelevant, right? It only matters that we only expect numbers as input, and so Number(x) is shorter than parseInt(x, 10) ...

On the (lack of) scale we're doing things, yes, the performance difference is largely irrelevant (the difference is measured in nanoseconds, which are billionths of a second)

Reply:
Keep in mind that using `Number` can result in some quirks.

```JS
parseInt("");    // returns NaN
parseInt(null);  // returns NaN
Number("");      // returns 0
Number(null);    // returns 0
```

I don't believe that this impacts us right now, but it is something to keep in mind.

Reply:
> Keep in mind that using `Number` can result in some quirks.

Javascript is built on quirks, like `isNaN` vs `Number.isNaN`, it's great stuff